derived_products/broadband_fluxes/dependencies.inc

```diff
@@ -13,7 +13,7 @@ $(OBJS)/two_rt_lw.o: $(OBJS)/kinds.o
 $(OBJS)/two_rt_lw_iter.o: $(OBJS)/kinds.o
 $(OBJS)/two_rt_lw_ocastrndm.o: $(OBJS)/kinds.o
 $(OBJS)/two_rt_lw_sel.o: $(OBJS)/kinds.o
-$(OBJS)/two_rt_sw_bs.o: $(OBJS)/kinds.o
+$(OBJS)/two_rt_sw_bs.o: $(OBJS)/kinds.o $(OBJS)/bandsolve.o
```

Comment:
Where did this pop up from? Was something broken before or is this for a new feature?

Reply:
This one is curious. The file invokes the dependency at https://github.com/ORAC-CC/orac/blob/c3cacad2266d9666a502bd29c5a67e9e84d2f75a/derived_products/broadband_fluxes/bugsrad/two_rt_sw_bs.f#L19 but `make depend` skips it. Thus, this is me manually adding it in to prevent parallel make from failing.